render.yaml

```diff
@@ -4,13 +4,13 @@ services:
 - type: web
   name: blockipedia
   env: static
-  buildCommand: cd frontend; yarn; yarn build
+  buildCommand: make build-frontend
   staticPublishPath: ./frontend/public
   pullRequestPreviewsEnabled: true  # see https://render.com/docs/pull-request-previews
   envVars:
   - key: GATSBY_CONTRACT_ADDRESS
     value: test.blockipedia.testnet
-    previewValue: dev-1654392200283-45248039192531  # only applicable to PreviewEnvironments
+    previewValue: dev-1654959502639-42307933260659  # only applicable to PreviewEnvironments
```

Comment:
I think we should not check in this dev contract. This seems more fitting for testing some contract function locally. If needed, we can always create another sub contract on testnet.

Reply:
this was actually my attempt to experiment [Render's Preview Environments feature](https://render.com/docs/preview-environments). This `previewValue`is only applicable when using this feature. However, I've decided to not proceed with this feature, so I'll remove this line.